README.md

```diff
@@ -224,10 +233,11 @@ Popular command line options are:
 - `--folder ID` - filter tests in specified folder.
 - `--environment-id` - run your tests using this environment. Otherwise it will use your default environment
 - `--conflict OPTION` - use the `abort` option to abort any runs in progress in the same environment as your new run. use the `abort-all` option to abort all runs in progress.
-- `--fg` - results in the foreground - rainforest-cli will not return until the run is complete. This is what you want to make the build pass / fail dependent on rainforest results
+- `--bg` - creates a run in the background - rainforest-cli will not return immediately. Does not use if you want to make the build pass / fail dependent on rainforest results.
```

Comment:
It actually will return immediately. :P I'd also reword it from build pass/fail to exit with error code, because this wording implies usage in ci environment.